sdk/nodejs/cmd/run/run.ts

```diff
@@ -22,8 +22,10 @@ import * as ini from "ini";
 import * as semver from "semver";
 import { ResourceError, RunError } from "../../errors";
 import * as log from "../../log";
+import * as opentelemetry from "@opentelemetry/api";
 import * as stack from "../../runtime/stack";
 import * as settings from "../../runtime/settings";
+import * as tracing from "./tracing";
```

Comment:
Consider lazy-load here to not pull any more dependencies than is needed especially when we're not tracing? Perhaps this is a small library but it seems there's a lot of transitive dependencies via opentelemetry code. Not sure how large they are though and how many files.

Reply:
I'm not sure its entirely avoidable: other files throughout the runtime inevitably call `getTracer().newSpan()` regardless of whether OpenTelemetry is enabled or not. That's the API pattern OTel expects -- everything is a no-op unless initialized, so client code should look the same whether or not tracing is occurring.

We could add a shim over the OTel API to create our own no-import, no-op trace API but we'd probably sacrifice a lot of conveniences of the OTel API. Happy to discuss further if you're unsatisfied with my justification :)